tests/test_usecase.py

```diff
@@ -189,10 +189,23 @@ def tearDown(self):
 
 class TestAlone(CommandTestCase):
 
-    def test_alone_misses_command(self):
-        with self.assertRaises(FakeSystemExit):
+    def test_alone_prints_help(self):
```

Comment:
We could keep this for python 3 only. (With a modification for the error message printed before the help.)